src/views/ont/ONTRunIndex.vue

```diff
@@ -104,6 +106,15 @@ export default {
       this.$router.push({ path: `/ont/run/${runId || 'new'}` })
     },
     ...mapActions(['fetchOntRuns']),
+    async fetchRuns() {
+      const page = { size: this.page_size.toString(), number: this.page_number.toString() }
```

Comment:
Again in a lot of places we are doing exactly the same thing. Should page be an object we can import and standardise across the app.

Reply:
Agreed, I have moved this behaviour into the queryParamsHelper, it also means we don't need to import lots of single use data in the setup.